Formula/kubernetes-helm.rb

```diff
@@ -45,5 +48,6 @@ def install
 
     version_output = shell_output("#{bin}/helm version --client 2>&1")
     assert_match "GitTreeState:\"clean\"", version_output
+    assert_match stable.instance_variable_get(:@resource).instance_variable_get(:@specs)[:revision], version_output if build.stable?
```

Comment:
As far as I know nobody fixed this, and if that's the case, `build.xyz` doesn't work in `test do`. You'd need to use `if.stable?`.